css/src/components/button.scss

```diff
@@ -0,0 +1,245 @@
+@use 'sass:map';
+
+$button-color: $text !default;
+$button-background-color: $body-background !default;
+
+$button-large-font-size: $font-size-6 !default;
+$button-small-font-size: $font-size-8 !default;
+
+$button-border-color: $text-subtle !default;
+$button-border-width: $control-border-width !default;
+$button-border-radius: $radius-small;
+
+$button-padding-vertical: calc(0.375em - #{$button-border-width}) !default;
+$button-padding-horizontal: 0.75em !default;
+
+$button-hover-color: $text !default;
+$button-hover-border-color: $default-hover !default;
+$button-hover-background-color: $default-hover-invert !default;
+
+$button-disabled-background-color: $body-background-medium !default;
+$button-disabled-shadow: none !default;
+$button-disabled-opacity: 0.5 !default;
+
+$button-icon-spacing: 0.375em !default;
+$button-font-weight: $weight-semibold !default;
+
+@if $features-button {
```

Comment:
Is this still needed? Looks like some of the file is not enclosed within. Should the variables also be inside? Thinking individual file imports would be easier to manage than this.

Reply:
Yeah, since we can't include them in index files, I think I'm going off the idea too :(.